proto/api/v2/compliance_results_stats_service.proto

```
@@ -16,6 +16,7 @@ message ComplianceScanStatsShim {
   repeated ComplianceCheckStatusCount check_stats = 2;
   google.protobuf.Timestamp last_scan = 3;
   string scan_config_id = 4;
+  ComplianceDataState data_state = 5;
```

Buf (1.72.0)

[error] cannot find `ComplianceDataState` in this scope

(COMPILE)